app/code/Magento/CatalogDataExporter/Model/Provider/Product/ProductOptions/OptionValueUid.php

```diff
@@ -1,38 +0,0 @@
-<?php
```

Comment:
app/code/Magento/ProductVariantDataExporter/Test/Integration/AbstractProductVariantsTest.php
Please, delete usage of this class

Reply:
Done